src/components/nav/sidebarItem.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { FC } from 'react'
+import * as Sidebar from './style'
+import { SidebarLink } from './sidebarLink'
+import { Icon } from 'react-icons-kit'
+
+const genres = [
+  'action',
+  'adventure',
+  'animation',
+  'comedy',
+  'horror',
+  'family',
+  'crime',
+  'disaster',
+  'documentary',
+  'drama',
+  'eastern',
+  'fan-film',
+  'fantasy',
+  'film-noir',
+  'history',
+  'holiday',
+]
+
+
+export const SidebarItemGroup: FC<{
+  toggleCategories(): void,
+  size: number,
+  icon: any,
+  label: string,
+  categoryState: string,
+  handleCategories(state: string): void
+}> = ({ handleCategories, toggleCategories, icon, size, label, categoryState }) => (
+  <>
+    <Sidebar.SidebarItem active={false} onClick={toggleCategories}>
+      <Icon icon={icon} size={size} />
+      <Sidebar.SidebarSpan>{label}</Sidebar.SidebarSpan>
+    </Sidebar.SidebarItem>
+
+    <Sidebar.SidebarNestedItemsWrapper
+      pose={categoryState === 'opened' ? 'opened' : 'closed'}
+      onPoseComplete={() =>
+        categoryState === 'closed' ? handleCategories('none') : null
+      }
+    >
+      {categoryState !== 'none' && (
+        <>
+          {genres.map((item, key) => (
+            <SidebarGroupItem
+              exact={false}
+              key={key}
+              label={item}
+              to={`/categories/${item}`}
+            />
+          ))}
+        </>
+      )}
+    </Sidebar.SidebarNestedItemsWrapper>
+  </>
+)
+
+interface SidebarItemProps {
+  to: string;
+  exact: boolean;
+  size: number;
+  label: string;
+  icon: unknown;
```

Comment:
Can you find a substitute for `unknown`?

Reply:
The same for last one too , It's defined `any` in the module I've imported it from.